guides/v2.3/graphql/functional-testing.md

```diff
@@ -170,3 +170,123 @@ $registry->register('isSecureArea', false);
 
 A fixture rollback filename should correspond to the original fixture filename postfixed by `_rollback` keyword. (For example, `virtual_product_rollback.php`.) 
 Magento provides fixtures in the `dev/tests/integration/testsuite/Magento/<ModuleName>/_files` directory. Use these fixtures whenever possible. When you create your own fixture, also create a proper rollback.
+
+## Expected exceptions
+Sometimes we need to cover a negative scenarious in our test and make sure that our functionality proper responses on them. There are two ways how you can define the expected exception message:
```

Comment:
Please, rephrase the description slightly by removing "we, our etc" occurrences. In case of documentation, such writing style might be a bit confusing. In result, it will be something like: "In cases when it's necessary to cover negative scenarios ... There are two ways how the expected exception message can be defined".

Reply:
**fixed**

Reply:
Or to minimize further, it could simply be.. "Sometimes you need to cover a negative scenario in your test to ensure..."

It's okay to use "you". Just keep the sentences simple and concise.

Reply:
Hi, @shrielenee ! Thank you for your notice.

Could you please clarify if this version is ok?

**Current version:** 
```In cases when it's necessary to cover negative scenarios in the test and make sure that testing functionality proper responses on them use the next ways to define the expected exception message:```

Thanks!

Reply:
I don't understand what this phrase means --- "make sure that testing functionality proper responses on them"

I would reword it similarly to...

> When it is necessary to cover negative test scenarios and ensure that testing elicits proper responses, define the expected exception message:

Reply:
But this PR needs a copyedit for language/grammar when the technical content is done.

Reply:
> I would reword it similarly to...

**implemented**

Thank you, @shrielenee !


> But this PR needs a copyedit for language/grammar when the technical content is done.

@rogyar , do you have any suggestions to the technical content?

Many thanks!

Reply:
Ping me when technical content is done and I'll copyedit.